pom.xml

```diff
@@ -175,4 +190,16 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
-</project>  
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.jenkins-ci.tools</groupId>
+                <artifactId>maven-hpi-plugin</artifactId>
+                <configuration>
+                    <compatibleSinceVersion>2.0</compatibleSinceVersion>
```

Comment:
This is the flag that the data format has changed and you cannot downgrade after upgrading without a backup